### PR TITLE
Feature/new home slider section

### PR DESCRIPTION
**Pull request by weberjavi**
This PR implements home redesign CW sections slider:
Slides designs could be found here:
https://projects.invisionapp.com/d/main#/console/16156416/335004751/preview

`IPSTACK_GEOLOCATION_API_KEY` env variable should be added to be able to get geolocator data.

### CODE REVIEW

File: app/javascript/app/styles/ui-extends.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  font-family: $font-family-1;
+  letter-spacing: 0;
+}
+
+// h1
+%title {
+  @extend %texts-defaults;
+
+  font-size: $font-size-xxl-large;
+  // line-height: $title-line-height;
```
Comment (aabdaab) [inline app/javascript/app/styles/ui-extends.scss]: remove comments pls 🙏

Comment (weberjavi) [reply app/javascript/app/styles/ui-extends.scss]: 🤦‍♂️

**Comment by weberjavi** [comment]
@aabdaab deleted commented code and updated the case in which geolocation is not provided (disable the `Explore your country` button and changed dropdown placeholder text).

A change is needed on `Carousel` component to address the sliding issue.

**Comment by weberjavi** [comment]
I was thinking about updating `cw-components` in another PR just to have this merged in the common branch to avoid conflicts. But it is alright with me to update component right away and change the version on this branch.
👍

**Pull request merged**